pybamm/solvers/processed_variable.py

```diff
@@ -6,6 +6,7 @@
 import numpy as np
 import pybamm
 import scipy.interpolate as interp
+from scipy.interpolate import RegularGridInterpolator
```

Comment:
no need to import here as you do `interp.RegularGridInterpolator` later

Reply:
Sure!